templates/start.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <form action="/start/" method="post">
+        <input type="text" name="name" id="name" placeholder="Имя">
+        <input type="text" name="mail" id="mail" placeholder="Почта">
```

Comment:
Тря проверки, что действительно был введен `email`
```suggestion
        <input type="email" name="mail" id="mail" placeholder="Почта">
```